features/virtual-pressure-sources.yml

```diff
@@ -5,6 +5,6 @@ group:
   - compute-pressure
   - webdriver
 compat_features:
-  - webdriver.commands.CreateVirtualPressureSource
-  - webdriver.commands.DeleteVirtualPressureSource
-  - webdriver.commands.UpdateVirtualPressureSource
+  - webdriver.classic.CreateVirtualPressureSource
```

Comment:
BCD moved the webdriver keys to a new namespace, to make some space for webdrivder bidi. No impact on data.